appmetrics/app_metrics.go

```diff
@@ -146,7 +148,7 @@ func (am *AppMetrics) getAppData(guid string) (*App, error) {
 		// If it exists in the cache, use the cache
 		app = am.Apps[guid]
 		timeToGrab := (time.Now().Add(-time.Duration(am.grabInterval) * time.Minute)).Unix()
-		if !app.ErrorGrabbing && app.updated > timeToGrab && !app.GrabAgain {
+		if !app.ErrorGrabbing && app.updated > timeToGrab {
```

Comment:
Do you mind to explain why we removed this `GrabAgain` attr?
Should we remove it from the app object as well?

Reply:
Oh yeah this attribute does not make sense, cache will be refreshed at regular intervals, and this attribute was effectively always true, so we never grabbed from the cache. cf PR description.
Yup I'll remove it from the struct.